docs/security.md

```diff
@@ -188,6 +202,11 @@ The following table describes the different options available for configuring th
   </td>
   <td>2.2.0</td>
 </tr>
+<tr>
+  <td><code>spark.network.crypto.authEngineVersion</code></td>
+  <td>1</td>
+  <td>Version of AES-based RPC encryption to use. Valid versions are 1 or 2. Version 2 is recommended. Added in 4.0.0</td>
```

Comment:
`Added in 4.0.0` looks like a typo. This will break this HTML table. In other words, we need to have two lines of `<td></td>`. One for description and the other for version. Please remove ` Added in 4.0.0` from here and create a new line, `<td>4.0.0</td>`, in order to follow the other configuration's style.

Reply:
Good catch. Fixed.